requirements/ci.in

```diff
@@ -1,6 +1,6 @@
 # Requirements for running tests in Travis
 -c constraints.txt
 
-codecov                   # Code coverage reporting
+coverage                  # Calculate code coverage of tests.
```

Comment:
`ci.in` should only pull in tox and tox-battery -- tox manages its own virtualenv where it pulls in `coverage` (via `test.txt` I believe).

Reply:
(so this line should just be deleted)